_config/users.yml

```diff
@@ -66,6 +66,13 @@
   twitterUsername: dupliaka
   employedBy: Red Hat
 
+  userId: michaltomco
```

Comment:
`- ` missing in front of userId. That breaks yml, causing the website to fail.
Once fixed, this is good to merge.

Reply:
that dash indicates that this is the start of a new element in the users array

Reply:
I was just going to build it locally. Thanks!

Reply:
Thank you for noticing Geoffrey!